Software/Development/SemesterProject/App/openLoopControlManager.c

```diff
@@ -0,0 +1,80 @@
+const float vfConst = 0.0185185; //1/54Hz
+
+
+/*
+ * mapFloat maps the value from one reference to another.
+ * Assuming only floats and minimum value of 0 for x and y.
+ */
+float mapFloat(float x, float xMax, float yMax)
+{
+    return x * yMax / xMax;
+}
+
+int16 mapInt(int16 x, int16 xMax, int16 yMax)
+{
+    return x * yMax / xMax;
+}
+
+
+
+void setConstantDutyFromPot(void)
+{
+    setAllDuties(mapFloat(1.2, 3.3, readMaxDuty())); //mapFloat(readADC(), readMaxADC(), readMaxDuty()));
+}
+
+
+//
+//This function is called at switching frequency so the duty needs to get updated in every iteration.
+//
+void setFrequencyFromPot(void)
```

Comment:
As this function will be called from controlTask, I think it should have a more general name, like open loop control, vfopenloop control or whatever. I realized that revieweing nicolais work in control task and the call to this function seems weird because is not providing enough info, IMO

Reply:
Substituted by openLoopVFControl( ).